src/scrape_up/Indiantrekking/trek.py

```diff
@@ -0,0 +1,68 @@
+from bs4 import BeautifulSoup
+from playwright.sync_api import sync_playwright
```

Comment:
Hey @Anshika14528, can you do it without any `Webkit`. like `sync_playwright`
Use Beautiful Soup Only.

Reply:
@nikhil25803 sir , I have updated it  to use requests module instead of sync_playwright 
 Can you please review it again?